src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -274,7 +277,7 @@ public void buttonClearDatePickerContestDateClicked() {
 
   public void buttonCvrFilePathClicked() {
     FileChooser fc = new FileChooser();
-    fc.setInitialDirectory(new File(System.getProperty("user.dir")));
+    fc.setInitialDirectory(new File(FileUtils.getUserFolder()));
```

Comment:
Damn.  Actually you are correct, but not because of this code here.  The text cells in the GUI are "observable" and whenever the user clicks 'add' to add a new cvr record, the GUI code calls the getter getFilePath() which as written now always returns the user path.  I'm going to fix this by renaming the functions.  

To code you reference here, this will set the initial folder location when you are browsing to select a cvr file to the user folder which is I believe desired behavior.  The file chooser will always return an absolute path, which is also I believe desired behavior.  If a user wants to save a relative path they will need to enter it manually into the text edit box.

Reply:
Oh, yeah, sorry, I wasn't trying to reference this particular line. GitHub just doesn't let me comment inline on a line that you didn't modify.

I'm a little confused about the intention of these directory changes, then. I thought part of the point was to facilitate sharing across environments by making it easier to set relative paths in the config. So you're saying that the file picker (for the output directory and the CVR source files) will still return an absolute path, but if the config already has a relative path and I don't modify it, it won't get changed to absolute when I save the config?

Reply:
That is correct:  File picker still returns an absolute path.  Existing paths will not be modified.

This point is still "This _enables_ portable user, debug, and regression test configurations, more intuitive path behavior in general"

I will be using this feature as I clean up unit test data, and George, Caleb and our "power users" can use it to share data as well.